### PR TITLE
Add catch-all routes that cover embedded work package details

### PR DESCRIPTION
**Pull request by tilt**
https://www.openproject.org/work_packages/13599

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -519,4 +519,8 @@
   match '/:controller(/:action(/:id))'
   match '/robots' => 'welcome#robots', :defaults => { :format => :txt }
   root :to => 'account#login'
+
+  # states managed by client-side routing on work_package#index
+  get '/work_packages/:id/*state' => 'work_packages#index'
+  get 'projects/:project_id/work_packages/:id/*state' => 'work_packages#index'
```
Comment (ulferts) [inline config/routes.rb]: Why not nest those within the blocks for work_packages respective projects/work_packages?

**Comment by tilt** [comment]
@ulferts @myabc Updated this, so let's see if the build will pass

**Comment by myabc** [comment]
@tilt much better!

**Comment by ulferts** [comment]
:+1:

**Pull request merged**